packages/middleware-sdk-transcribe-streaming/src/signer.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { SignatureV4 } from "./signer";
+import { HttpRequest } from "@aws-sdk/protocol-http";
```

Comment:
mock HttpRequest.isInstance to return true

Reply:
Will postpone this change to later according to offline discussion.